.reuse/dep5

```diff
@@ -1,7 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: librepcb
 Source: https://librepcb.org/
-Files-Excluded: libs/parseagle/specs/eagle*dtd
 Comment:
  Exclude libs/parseagle/specs/eagle_7.5.0.dtd as it is released
  under CC-BY-ND-3 and not required for building or installing.
```

Comment:
I just realized this comment is related to `Files-Excluded` which was removed, so I think this comment should be removed too.

Reply:
Updated.